registry/sql-registry/registry/db_registry.py

```diff
@@ -94,7 +94,7 @@ def get_projects(self) -> List[str]:
             ret = self._fetch_helper(query)
         else:
             ret = self.conn.query(
-            f"select qualified_name from entities where entity_type=%s", str(EntityType.Project))
+            f"select 5 from entities where entity_type=%s", str(EntityType.Project))
```

Comment:
Was this for testing only?

Reply:
Yes, sorry for this, I made a temporary modification during sandbox testing in order to see the data.